File: python/appsettings.py
import os


class AppSettings:
    def __init__(self):
        self.token = ""
        self.app_id = ""
        self.supergoon_games_server_id = ""
        self.github_access_token = ""


application_settings = AppSettings()


def initialize_app_settings():
    application_settings.token = os.getenv("DISCORD_BOT_TOKEN", "")
    application_settings.app_id = os.getenv("DISCORD_APP_ID", "")
    application_settings.supergoon_games_server_id = os.getenv("DISCORD_SUPERGOON_GUILD_ID", "")
    application_settings.github_access_token = os.getenv("GITHUB_ACCESS_TOKEN", "")

File: go/appsettings.go
package main

import "os"

type AppSettings struct {
	Token             string
	AppID             string
	GuildID           string
	GithubAccessToken string
}

var ApplicationSettings AppSettings

func InitializeAppSettings() {
	ApplicationSettings.Token = os.Getenv("DISCORD_BOT_TOKEN")
	ApplicationSettings.AppID = os.Getenv("DISCORD_APP_ID")
	ApplicationSettings.GuildID = os.Getenv("DISCORD_SUPERGOON_GUILD_ID")
	ApplicationSettings.GithubAccessToken = os.Getenv("GITHUB_ACCESS_TOKEN")
}
